.github/workflows/release.yml

```diff
@@ -17,7 +17,6 @@ jobs:
         with:
           node-version: 16.x
       - run: npm install
-      - run: npm install -g npm-run-all
```

Comment:
why?

Reply:
Noted in the PR description:

> adds npm-run-all package to devDependencies and removed from workflows since it's not needed there now

Generally, if a package is used in script commands, it should be included in `devDependencies`